soft4pes/control/mpc/controllers/rl_grid_mpc_curr_ctr.py

```diff
@@ -0,0 +1,131 @@
+""" Model-predictive controller for RL grid."""
+
+from types import SimpleNamespace
+import numpy as np
+from soft4pes.utils.conversions import dq_2_alpha_beta
+
+
+class RLGridMpcCurrCtr:
+    """
+    Model-predictive controller for RL grid. The controller will track the 
```

Comment:
The controller aims to track the grid current in the alpha-beta frame

Reply:
Solved in next commit